### DISCUSSION TITLE
(Double) click on item should open build

### DISCUSSION THREAD

**Discussion by Fabi755** [deszip/BitBot#35] [Feature requests]
At first thanks for this tool, and it's very nice that is now available on AppStore.

It's a little bit annoying to make a right click + click on `Open Build`. It took me a few days to understand that a right click is possible.

My expectation is that a click (or double click) will open the Bitrise webpage of clicked build item (like menu `Open Build`). 

What do you think? It's a good idea to change this?

**Comment by deszip** [comment]
Thanks for kind words @Fabi755. 
Interesting how you (me in this case) don't think of some behaviours like right click menu when you built it and know it's there from the beginning. Adding double click sounds interesting (I think single one could lead to lots of occasional clicks while scrolling). I should probably ask design guys :)

**Comment by deszip** [comment]
Hi @Fabi755 , pls check v1.1.0, double click now opens a build page in browser.
Thank you for suggestion.

**Reply by Fabi755** [reply]
Yes, it works fine! Thank you! 

One little thing I'm missing there:
For a more improved user feedback we should show some click and/or hover feedback. But I still have no idea how this could look in detail.

**Reply by Fabi755** [reply]
And make it sense to close the BitBot window after opening the build in browser?

**Reply by deszip** [reply]
Yep closing a popover after navigating to the build is a good idea, thanks.
Regarding UI feedback - will check with designer, maybe we can do smth about it.

Feel free to open issue/discussion in case you have more feedback.

Thanks.

**Comment by deszip** [comment]
@Fabi755 Version 1.2.0 already available via AppStore, popover is closing now if you navigate out of the app. Thanks for suggestion!
